tests/bootstrap.php

```diff
@@ -43,5 +44,5 @@
 
 Cake\Core\Configure::write('debug', true);
 
-Cake\Core\Plugin::load('CakeDto', ['path' => ROOT . DS, 'autoload' => true, 'bootstrap' => true]);
+Cake\Core\Plugin::getCollection()->add(new CakeDto\Plugin());
```

Comment:
Just to be clear: This is also available in 3.6 right?
The tests at least say it is - so all good for BC :)

Reply:
> Just to be clear: This is also available in 3.6 right?

Yeah, it's available in 3.6